stistools/response.py

```diff
@@ -0,0 +1,105 @@
+import math
+import numpy as np
+from astropy.io import fits
+
+import fit1d
+
+def response(calibration_array, normalization_array,
+             threshold=None,
+             function="spline3",
+             sample='*',
+             naverage=2,
```

Comment:
`naverage` doesn't get used below, which implies it should be 1 and anything else should raise a `NotImplementedError`.